### DISCUSSION TITLE
RegexConfig.txt

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1745] [Q&A]
I would like some guidance on where I am going wrong.
I tried putting some additional regex records in the "conf/RegexConfig.txt" file, but the result after indexing was unsuccessful.
So I took the original RegexConfig.txt file and put just one more line, duplicating the previous one like this:
BR_CAR_PLATE = \b((([A-Z]{3})(()|-)([0-9]{4}))|(Plates?( |:|: )([A-Za-z]{ 3})( )([0-9]{4})))\b
PLACA_MERCOSUL = \b((([A-Z]{3})(()|-)([0-9]{4}))|(Plates?( |:|: )([A-Za-z]{ 3})( )([0-9]{4})))\b

At the end of the indexing, several BR_CAR_PLATE records were located, but none with PLACA_MERCOSUL appeared, which I thought would be identical to BR_CAR_PLATE.

Do I need to configure something else?

tks

**Comment by lfcnassif** [comment]
I was able to reproduce the behavior you described with version 4.1.3. I'll try to investigate when I have available time.

**Reply by paulobreim** [reply]
I'm available for testing. by the way I have a T630 just for iped

**Comment by milcent-CVM** [comment]
I'm having the same issue in version 4.0.7, also reproduced in version 3.18.6.
I tried to add one Regex Pattern to find the "_Relator_" of the case after validating it in [https://regex101.com/](https://regex101.com/), like so:

`# Register REGEX below:`

**`RELATOR = (?:\n*)+(?:\s*\-?)?(?:relatora?)+(?:\s*:?\s*)(?:diretora?|presidente)?\s*(\w+(?:(?: +\w+)+))\b`**

`EMAIL , 0 , 0 , true = [0-9a-z][0-9a-z\+\.\_\%\-]{1,64}\@[0-9a-z\-]{2,64}(\.[0-9a-z\-]{2,25}){1,3}`

But IPED doesn't seem to consider it during processing.
I could try with some more trivial Regex, but given the example by @paulobreim, it seems the problem is with new added patterns.
Cheers

**Comment by lfcnassif** [comment]
Sorry, this went down in my task stack and I forgot about it. I just opened #1897 to don't forget it again. Unfortunately this week I'm in charge of my unit and maybe I will have time to look at this just in the next week, if no other developer or contributor takes a look at it before me.

**Reply by hpcnetworks** [reply]
Hi there!
I've got the sae issue on the latest version, 4.1.5.

**Reply by lfcnassif** [reply]
This should have been fixed on #1897. Please provide a sample regex and text file with the pattern to be matched that reproduces it on 4.1.5.
